js/_website/generate_jsons/generate.py

```diff
@@ -10,11 +11,18 @@ def make_dir(root, path):
     return os.path.abspath(os.path.join(root, path))
 
 def get_latest_release():
-    with open(make_dir(WEBSITE_DIR, "src/lib/json/version.json"), "w+") as j:
-        with open(make_dir(GRADIO_DIR, "package.json")) as f:
+    with open(make_dir(GRADIO_DIR, "package.json")) as f:
+        version = json.load(f)["version"]
+        with open(make_dir(WEBSITE_DIR, "src/lib/json/version.json"), "w+") as j:
             json.dump({
-                "version": json.load(f)["version"]
+                "version": version
                 }, j)
+        with open(make_dir(WEBSITE_DIR, "src/lib/json/wheel.json"), "w+") as j:
```

Comment:
I think we should add these generated files to gitignore?

Reply:
Everything in `src/lib/json` is gitignored